### DISCUSSION TITLE
path to logstash config  to send all output to ES

### DISCUSSION THREAD

**Discussion by Pancakesarusrex** [Security-Onion-Solutions/securityonion#5772] [Unsupported Versions] [answered]
what is the path to the logstash config to send all output to elasticsearch. i would like send a copy of all outputs to an external siem as well as elasticsearch.

**Comment by dougburks** [comment, marked as answer]
If you haven't already, please see:
https://docs.securityonion.net/en/2.3/logstash.html#forwarding-events-to-an-external-destination
